src/Compilers/Test/Core/DotNetCoreSdk.cs

```diff
@@ -22,6 +22,7 @@ public override string SkipReason
 
         static DotNetCoreSdk()
         {
+#if NETCOREAPP
```

Comment:
Why is this change necessary here? Trying to understand how this PR could have caused an error here.

Reply:
It doesn't really make sense to use the .NET SDK from .NET Framework. essentially if we are running tests from .NET Framework we do not want SDK-based tests to run.